powerdns/models/powerdns.py

```diff
@@ -583,7 +578,7 @@ def create_ptr(self):
                     'type': self.domain.type,
                 }
             )
-        elif self.auto_ptr == AutoPtrOptions.ONLY_IF_DOMAIN:
+        elif self.domain.auto_ptr == AutoPtrOptions.ONLY_IF_DOMAIN:
```

Comment:
what's the purpose of `ONLY_IF_DOMAIN` now? Is it possible to have record, which suffix isn't equal to it's domain name?